src/components/MultiBarOverlay/index.tsx

```diff
@@ -31,7 +31,8 @@ export const MultiBarOverlay: React.FC<Props> = ({ navigation }) => {
 
     const animationsList = animations.map((anim, idx) => animate(anim, {
       toValue: extrasVisible ? 1 : 0,
-      delay: idx * 150
+      delay: idx * 150,
+      useNativeDriver: true
```

Comment:
ReactNative doesn't support `useNativeDriver: true` for properties like: x, y, width, height, so should be false, please fix it then I can merge.